packages/trader/src/Modules/CFD/Containers/compare-accounts-content.tsx

```diff
@@ -0,0 +1,690 @@
+import React from 'react';
+import { Table, Text, ThemedScrollbars, Div100vhContainer } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { isDesktop, CFD_PLATFORMS, isLandingCompanyEnabled, PlatformContext } from '@deriv/shared';
+import { LandingCompany } from '@deriv/api-types';
+
+type TCFDAttributeDescriberProps = {
+    name: string;
+    counter: number | null;
+};
+
+type TFilterAvailableAccounts = (
+    landing_companies: LandingCompany,
+    table: TAccountsDescription[],
+    is_logged_in: boolean,
+    show_eu_related: boolean,
+    platform: string,
+    is_australian: boolean
+) => Array<{ [key: string]: string | React.ReactNode | undefined }>;
+
+type TAccountsDescription = {
+    attribute: string;
+    mt5: TDxTradeAccountsDescription & { financial_stp: string };
+    dxtrade: TDxTradeAccountsDescription;
+};
+
+type TDxTradeAccountsDescription = {
+    synthetic: string;
+    synthetic_eu: string;
+    financial: string;
+    financial_au: string;
+    financial_eu: string | React.ReactNode;
+    footnote: string | null;
```

Comment:
Same for here, instead of using type string | null use correct type for this

Reply:
Same goes for here.